src/_data/sites/en.yml

```
@@ -385,7 +385,7 @@ donate_page:
   description: >
     ESLint relies on donations for ongoing maintenance and development.
     There are currently BACKER_COUNT companies, organizations, and individuals
-    are donating MONTHLY_DONATIONS each month to help us out.
+    who are donating MONTHLY_DONATIONS each month to help us out.
```

I think we can make this a bit simpler. 

```suggestion
    donating MONTHLY_DONATIONS each month to help us out.
```